.github/workflows/rust.yml

```diff
@@ -27,8 +27,60 @@ concurrency:
 env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: full
+  MIN_SUBMITTERCOVERAGE_PERCENTAGE: 1
 
 jobs:
+  submitter-coverage:
+    runs-on: buildjet-8vcpu-ubuntu-2204
```

Comment:
does this need 8vcpu or can we go lower? available images: https://buildjet.com/for-github-actions/docs/runners/hardware#amd

note that adding one more job that requires a buildjet agent means we need to boost our CI capacity ASAP (or we accept slightly longer waits for PRs)